nodes/AugAssign.py

```diff
@@ -21,10 +20,11 @@
     if necessary (as in BinOp).
 
 
-Example:
-    - target  -> [Name(id='x', ctx=Store())]
-    - value   -> Num(n=1)
-    - op      -> Add()
+Example 1:
```

Comment:
You shouldn't add the numbering here unless you plan to add another example.

Reply:
(The same below as well.)

Reply:
I have made changes accordingly.